test/utils/permit2.ts

```diff
@@ -1,11 +1,19 @@
 import { Wallet } from 'ethers'
 import { AllowanceTransfer, PermitSingle } from '@uniswap/permit2-sdk'
 import { Permit2Permit } from '../../src/utils/permit2'
-import { PERMIT2_ADDRESS } from './addresses'
+import { PERMIT2_ADDRESS, ROUTER_ADDRESS } from './addresses'
+import { ethers } from 'ethers'
+
+const TEST_DEADLINE = '3000000000000'
```

Comment:
Is this the value that will be put into txs through the frontend? If so we should optimise it to have as many 0s as possible

Reply:
No, the front end will have a dynamic value x seconds into the future. I have a static value for tests, so that we won't keep updating test encoding data for every PR.